### PR TITLE
CANS-1014: Swap item 7 & 8

### PR DESCRIPTION
**Pull request by LeonidMarushevskyi**
## JIRA

https://osi-cwds.atlassian.net/browse/CANS-1014

## Tests
- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters and static analyzers for the project (SonarQube, eslint, rubocop, etc).
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.

### CODE REVIEW

File: tools/instrument-parser/csv/cans_items.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿Item_ID,CANS_6_21_ID,CANS_0_5_ID,Item_Name ,Item_Abbr,Description,Rate_0,Rate_1,Rate_2,Rate_3,Rate_No,Rate_Yes,Rate_NA,Confidential_by_Default,Questions,Supplemental,0-5,6-21,Need,Strength,Background_Needs,Outcome_Malleable,Danger_Self,Danger_Others
+Item_ID,CANS_6_21_ID,CANS_0_5_ID,Item_Name ,Item_Abbr,Description,Rate_0,Rate_1,Rate_2,Rate_3,Rate_No,Rate_Yes,Rate_NA,Confidential_by_Default,Questions,Supplemental,0-5,6-21,Need,Strength,Background_Needs,Outcome_Malleable,Danger_Self,Danger_Others
```
Comment (denys-davydov) [inline tools/instrument-parser/csv/cans_items.csv]: Concerned about this line update when no changes are made. However, if it works as expected then it's fine.

File: src/main/resources/liquibase/cans_database_demo_master.xml [hunk 2]
```diff
@@ -16,4 +16,7 @@
   <include file="ca/2018-12-28_update_ca_instrument_i18n.xml" relativeToChangelogFile="true"/>
 
   <include file="ca/2019-02-22_update_ca_instrument_i18n.xml" relativeToChangelogFile="true"/>
+
+  <include file="ca/2019-02-27_update_ca_instrument.xml" relativeToChangelogFile="true"/>
+  <include file="ca/2019-02-27_update_ca_instrument_i18n.xml" relativeToChangelogFile="true"/>
```
Comment (denys-davydov) [inline src/main/resources/liquibase/cans_database_demo_master.xml]: I am not sure we need to update instrument i18n in this situation. Could you please confirm the update is needed?

Comment (LeonidMarushevskyi) [reply src/main/resources/liquibase/cans_database_demo_master.xml]: i18b was changed so I will keep this row as is.

File: tools/instrument-parser/csv/cans_items.csv (hunk 1 above)
Comment (LeonidMarushevskyi) [reply tools/instrument-parser/csv/cans_items.csv]: Don't see any difference and can confirm that assessment works as expected. So propose to keep this change.

**Pull request merged**